src/YAMLValidator.Tests/ValidatorTest.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Json.Schema;
+using Microsoft.PowerPlatform.PowerApps.Persistence;
```

Comment:
Please add "YamlValidatorTests" so it becomes: Microsoft.PowerPlatform.PowerApps.Persistence.YamlValidatorTests;